src/cow-react/modules/limitOrders/containers/QuoteResolver/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { useSetAtom } from 'jotai'
+import { limitOrdersQuoteAtom } from '@cow/modules/limitOrders/state/limitOrdersQuoteAtom'
+import { useEffect } from 'react'
+import { useQuoteRequestParams } from '@cow/modules/limitOrders/containers/QuoteResolver/hooks/useQuoteRequestParams'
+import { getQuote } from '@cow/api/gnosisProtocol'
+
+export const QuoteResolver = () => {
```

Comment:
Does it make sense to have the resolver inside an `updater` directory (`limitOrders/updater`) and maybe call it `QuoteUpdater` since components that return null and just do some side effects in the app, we call them updaters

Reply:
Thank you!
Yes, this is updater